apps/src/codebridge/Console/RightButtons.tsx

```diff
@@ -10,22 +11,31 @@ interface RightButtonsProps {
   clearOutput: () => void;
 }
 
+const tooltipProps = {
+  text: 'Clear Console',
```

Comment:
I noticed some strings are translatable and some aren't yet. Want me to add these? To common?

Reply:
Yes please! Since it's codebridge specific it can go in codebridge I18n (if it already exists in common feel free to reuse). We are updating to translatable strings as we go, and have a catch-all task to translate all the remaining ones when we have time for it: [CT-631](https://codedotorg.atlassian.net/browse/CT-631)